### DISCUSSION TITLE
How to make pull request for new language (Gujarati, Punjabi, Tamil)

### DISCUSSION THREAD

**Discussion by GeneralCheesecake** [Laravel-Lang/common#34] [Q&A] [answered]
Hello,

I am a programmer for a company, and we work with people in India with a Laravel application. 
For them we translated the following form validation files, which were not in this repo in Gujarati, Punjabi and Tamil:

- auth.php
- pagination.php
- passwords.php
- validation.php

We also had the custom json file with translations of sentences in our app, but I feel like those lines are not relevant for this repository. I felt like contributing these (new) translations to this repository, but I am quite new with working on a repository like this. I found the manual for adding a new language [(this one)](https://laravel-lang.github.io/lang/), but I do not understand how to make a pull request. 

On my local machine, I cloned the Laravel-Lang/lang repository and added the files for the Gujarati translation in the folder names 'gu'. I put in the files and saved them. But what is the next step? Do I commit and push? Because I can't find anywhere if I am supposed to make the changes on the master branch or a new branch. 

So what do I have to do now? Thank you in advance!

**Comment by caouecs** [comment, marked as answer]
Hello,
it's very simple : https://laravel-lang.github.io/lang/contributing-to-dev.html ;)

**Comment by GeneralCheesecake** [comment]
It seems like I just was not familiar with how forks worked. I am only used to working with branches and pull requests between branches, so pull requests between forks was a new concept to me. But I got my pull request to work, so hopefully that will be allright!
